src/main/java/com/coreos/jetcd/internal/impl/LeaseImpl.java

```diff
@@ -313,13 +322,12 @@ private LeaseKeepAliveRequest newKeepAliveRequest(long leaseId) {
   }
 
 
-  private class KeepAliveListenerImpl implements KeepAliveListener {
+  private static class KeepAliveListenerImpl implements KeepAliveListener {
```

Comment:
I was playing static bug checker. It complains about the nested class not being static. It seems to be the best practice in java is to have inner static class.
https://stackoverflow.com/questions/19825956/java-inner-class-access-and-best-practices